sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md

```diff
@@ -1,12 +1,38 @@
 # Release History
 
-## 1.0.0 (Unreleased)
+## 1.0.0 (2020-10-30)
+
+### New Features
 
 - Regenerate protocol layer from service API version 2020-10-31
 - Update service API version to use service API version 2020-10-31 by default
-- Add optional parameters for TraceParent and TraceState to all service request APIs to support distributed tracing
-- Renamed model type "ModelData" to "DigitalTwinsModelData" to make type less generic, and less likely to conflict with other libraries
-- Renamed model type "RequestOptions" to "DigitalTwinsRequestOptions" to make type less generic, and less likely to conflict with other libraries
+
+
+### Breaking changes
+
+Note that these breaking changes are only breaking changes from the preview version of this library.
+
+- Replace all `Response<string>` and `Pageable<string>` APIs with `Response<T>` and `Pageable<T>` respectively
+- Rename `CreateDigitalTwin`, `CreateRelationship` and `CreateEventRoute` APIs to `CreateOrReplaceDigitalTwin`, `CreateOrReplaceRelationship` and `CreateOrReplaceEventRoute` respectively
+- Renamed model type `ModelData` to `DigitalTwinsModelData` to make type less generic, and less likely to conflict with other libraries
+- Renamed model type `EventRoute` to `DigitalTwinsEventRoute` to make type less generic, and less likely to conflict with other libraries
+- `EventRoute` (now `DigitalTwinsEventRoute`) object ctor now requires filter
+- Remove `UpdateOperationsUtility` and replace it with a direct dependency on [JsonPatchDocument](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/src/JsonPatchDocument.cs) from Azure.Core
+- Remove `WritableProperty` since service no longer returns that type
+- Remove `MaxItemCount` parameter as an option for `GetEventRoutes` APIs since users are expected to provide page size in pageable type's .AsPages() method instead
+- Remove Serialization namespace, moving its contents to the base Azure.DigitalTwins.Core namespace
+- Rename `DigitalTwinsModelData` field `DisplayName` to `LanguageDisplayNames` for clarity
+- Rename `DigitalTwinsModelData` field `Description` to `LanguageDescriptions` for clarity
+- Rename `DigitalTwinsModelData` field `model` to `dtdlModel`
+- Flatten `DigitalTwinsRequestOptions` so that each API takes in `ifMatch` and `ifNoneMatch` header directly
+- Rework `BasicDigitalTwin` and other helper classes to better match the service definitions. This includes renaming `CustomProperties` to `Contents`.
+- Add `messageId` as mandatory parameter for telemetry APIs. Service API version 2020-10-31 requires this parameter.
+
+
+### Fixes and improvements
+- Fix bug where `CreateDigitalTwin` and `CreateRelationship` APIs always sent ifNoneMatch header with value "*" making it impossible to replace an existing entity
```

Comment:
Should consistently use punctuation in each bullet for a cleaner, professional look.